stacks-signer/src/main.rs

```diff
@@ -81,6 +81,13 @@ fn write_chunk_to_stdout(chunk_opt: Option<Vec<u8>>) {
 fn spawn_running_signer(path: &PathBuf) -> SpawnedSigner {
     let config = GlobalConfig::try_from(path).unwrap();
     let endpoint = config.endpoint;
+    // TODO: check if config.node_host is in a given list of possible local hosts and only if it now, then display the message?
+    warn!(
```

Comment:
I do like the idea of printing this if the config.node_host is not a localhost for sure. Makes it cleaner and less likely someone will ignore it :D Thanks for doing this!

Reply:
The `regtest-env` setup sees it as `stacks-node:20443`. Should I include `stacks-node` or other names on the checker besides `localhost`?

Reply:
Hm I don't like the idea of special casing too much around names beyond local host...perhaps instead just print the warning at all times. We can always revisit this later.